### PR TITLE
Add tests for datascience notebooks

### PR DESCRIPTION
**Pull request by rkpattnaik780**
Add tests for Datascience notebooks

## Description

This PR adds tests for Datascience notebooks for Python v3.8 and v3.9. The tests evaluate the following python packages:

* Numpy
* Pandas
* Scipy
* Scikit-learn
* Matplotlib
* Kafka-python
* Boto3

Addresses #180 

## How Has This Been Tested?

Run Jupyter Minimal Notebooks for both the versions of Python, i.e 3.8 and 3.9. Run the test notebooks in the relevant environment. The tests should execute with all passing.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: jupyter/datascience/ubi8-python-3.8/test/test_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    "from kfp import LocalClient, run_pipeline_func_locally\n",
+    "\n",
+    "def get_major_minor(s):\n",
+    "    return '.'.join(s.split('.')[:2])\n",
+    "\n",
+    "class TestPythonVersion(unittest.TestCase):\n",
+    "    def test_version(self):\n",
+    "        expected_major_minor = '3.8'\n",
+    "        actual_major_minor = get_major_minor(python_version())\n",
+    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect version\")\n",
```
Comment (jstourac) [inline jupyter/datascience/ubi8-python-3.8/test/test_notebook.ipynb:41]: What about (similar for other such occurrences):

```suggestion
    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect Python version\")\n",
```

But, maybe the failing message is clear so my comment is irrelevant.

Comment (jstourac) [reply jupyter/datascience/ubi8-python-3.8/test/test_notebook.ipynb:41]: @rkpattnaik780 looks good, sorry for bugging :slightly_smiling_face:

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

Thanks :100:

**Pull request merged**